Tests/Kitura/KituraTest.swift

```diff
@@ -16,6 +16,7 @@
 
 import XCTest
 
+@testable import Kitura
```

Comment:
do you need testable annotation here? Isn't it for accessing non-public methods?